### ISSUE TITLE
cleanup: separate out and explain with comments the debug drawing functions

### ISSUE THREAD

**Issue by LevFendi** [Factorio-Access/FactorioAccess#117]

**Comment by ahicks92** [comment]
They need to be pulled out.  A lot of this code is inline.

Without the ability to see it, it's hard to modify a visualization even if explained, but it is reasonable to say "this visualizes lists, please call it here for the sighted people".  I'll also note that I understand visualization because I had some vision, but I'm 99%A sure that anyone who never had any is just going to be like wtf is this doing arg.

**Comment by LevFendi** [comment]
Yeah, cleanly separated and well-explained functions are necessary.

**Comment by ahicks92** [comment]
As hopefully constructive criticism, the word "maybe" in the title is what made me post my comment.
